cmd/server/flags.go

```diff
@@ -473,3 +473,14 @@ var flags = append([]cli.Flag{
 		Usage:   "Flag to decrypt all encrypted data and disable encryption on server",
 	},
 }, logger.GlobalLoggerFlags...)
+
+// If woodpecker is running inside a container the default value for
+// the datasource is different from running outside a container.
+func datasourceDefaultValue() string {
+	_, found := os.LookupEnv("WOODPECKER_IN_CONTAINER")
+	if found {
+		return "/var/lib/woodpecker/woodpecker.sqlite"
```

Comment:
This should be by default. Especially not in container.
https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard

Reply:
Yes, but the current default is current directory/woodpecker.sqlite and changing that would break e.g. development.
This option has been neglected.